components/authentication/base/group-sync/test-platform-ci-admins-rover-group.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: redhatcop.redhat.io/v1alpha1
+kind: GroupSync
+metadata:
+  name: test-platform-ci-rover-groups
+  annotations:
+    argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+spec:
+  providers:
+    - ldap:
+        ca:
+          key: ca.crt
+          kind: Secret
+          name: mtls-ca-validators
+          namespace: group-sync-operator
+        credentialsSecret:
+          name: konflux-ldap-sa
+          namespace: group-sync-operator
+        insecure: false
+        prune: true
+        rfc2307:
+          usersQuery:
+            baseDN: 'dc=redhat,dc=com'
+            derefAliases: never
+            scope: sub
+          groupNameAttributes:
+            - cn
+          tolerateMemberNotFoundErrors: true
+          tolerateMemberOutOfScopeErrors: true
+          groupUIDAttribute: dn
+          groupMembershipAttributes:
+            - uniqueMember
+          userNameAttributes:
+            - uid
+          groupsQuery:
+            baseDN: 'ou=adhoc,ou=managedGroups,dc=redhat,dc=com'
+            derefAliases: never
+            filter: (&(objectClass=rhatRoverGroup)(cn=test-platform-ci-admins))
+            scope: sub
+          userUIDAttribute: dn
+        url: 'ldaps://ldapfrac.corp.redhat.com'
```

Comment:
we use different ldap endpoint for internal clusters, just apply the following patch and it will use the internal endpoint for internal clusters:
```
diff --git a/components/authentication/k-components/ldap-url-patch/kustomization.yaml b/components/authentication/k-components/ldap-url-patch/kustomization.yaml
index fa6e8e5e..6fa3abd5 100644
--- a/components/authentication/k-components/ldap-url-patch/kustomization.yaml
+++ b/components/authentication/k-components/ldap-url-patch/kustomization.yaml
@@ -8,3 +8,9 @@ patches:
       kind: GroupSync
       group: redhatcop.redhat.io
       version: v1alpha1
+  - path: ldap-url-patch.yaml
+    target:
+      name: test-platform-ci-rover-groups
+      kind: GroupSync
+      group: redhatcop.redhat.io
+      version: v1alpha1

```